docs/containers.stories.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf } from '@storybook/html'; // eslint-disable-line import/no-extraneous-dependencies
+import { // eslint-disable-line import/no-extraneous-dependencies
+  withKnobs, boolean,
+} from '@storybook/addon-knobs';
+
+const stories = storiesOf('Containers', module);
+stories.addDecorator(withKnobs);
+
+stories.add('container', () => {
+  const withTitle = boolean('with-title', false) ? 'with-title' : '';
+  const isCenter = boolean('is-center', false) ? 'is-center' : '';
+  const isRight = boolean('is-right', false) ? 'is-right' : '';
```

Comment:
`is-center` and `is-right` are exclusive relationships 👐 
so radios is better 😆 
```js
const withTitle = boolean('with-title', false) ? 'with-title' : '';
  const alignment = radios('alignment', {
    default: '',
    'is-center': 'is-center',
    'is-right': 'is-right'
  }, '');
  // ...
  return `<div class="container ${selectedClasses.join(' ')} ${alignment}">
      <p class="title">Container</p>
      <p>Good morning. Thou hast had a good night's sleep, I hope.</p>
  </div>`;
```

![2018-12-05 14 24 27](https://user-images.githubusercontent.com/5305599/49491943-83d10b00-f899-11e8-9882-ccf733b870a4.png)